### PR TITLE
Login and signup pages

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
As part of this PR, the following pages have been created :

- login page    : https://login-and-singup.netlify.app/login.html
- signUp page : https://login-and-singup.netlify.app/signup.html

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <a class="nav-pill" href="index.html" ><img class="nav-pill-img" src="assets/images/home.png"></a>
+            <a class="nav-pill" href="wishlist.html" ><img class="nav-pill-img" src="assets/images/heart.png"></a>
+            <a class="nav-pill" href="cart.html"><img class="nav-pill-img" src="assets/images/bag.png"></a>
+            <a class="nav-pill active-nav-pill" href="login.html"><img class="nav-pill-img" src="assets/images/user.png"></a>
+        </nav>
+    </header>
+
+    <div class="flex-space-btwn flex-center space-around login-parent">
+        <div class="flex-column">
+            <img class="auth-landing-img" src="assets/images/auth-background.png">
```
Comment (foolhardy21) [inline signup.html]: you can give alt attribute here.

**Review by foolhardy21 — COMMENTED** [review]
rest looks good to me. good to merge after alt is added.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    <input class="form-control input-lg mg-tb-1 form-input" type="password" placeholder="Password"/>
+                    <div class="flex-layout flex-space-btwn width-100">
+                        <div class="flex-layout">
+                            <input type="checkbox"> 
+                            <label class="fs-13px">Remember me</label>
+                        </div>
+                        <a class="link-txt" href="">Forgot Password ?</a>    
+                    </div>
+                    <button class="checkout-btn">Login</button>
+                    <a class="link-txt" href="signup.html">Create new account >></a>
```
Comment (aayushjain-code) [inline login.html:53]: you can add an arrow icon for better Visual.
Very nice work go forward🚀

**Review by ApurvaSawant11 — APPROVED** [review]
UI looks good!

Just add some padding on the left and right of the forms and adjust the width accordingly.
Also, font-size for the form is quite small compared to the rest of the page.

File: signup.html (hunk 1 above)
Comment (samyukthasudhakar) [reply signup.html]: added alt attribute. Thanks for noticing !

File: login.html (hunk 2 above)
Comment (samyukthasudhakar) [reply login.html:53]: Sure will do that. Thank you 😄

**Comment by samyukthasudhakar** [comment]
> UI looks good!
> 
> Just add some padding on the left and right of the forms and adjust the width accordingly. Also, font-size for the form is quite small compared to the rest of the page.

Sure Apurva. Will do that 👍🏽

**Pull request merged**